client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -60,6 +60,70 @@ function PlaceholderPostEdit( props ) {
 	);
 }
 
+function RelatedPostsEditItem( props ) {
+	const previewClassName = 'related-posts__preview';
+
+	return (
+		<div className={ `${ previewClassName }-post` }>
+			{ props.displayThumbnails && props.post.img && props.post.img.src && (
+				<Button className={ `${ previewClassName }-post-link` } isLink>
+					<img src={ props.post.img.src } alt={ props.post.title } />
+				</Button>
+			) }
+			<h4>
+				<Button className={ `${ previewClassName }-post-link` } isLink>
```

Comment:
Why not use a regular link here?

Reply:
The same goes for the other button above.

Reply:
I agree and this impacts A11Y as well. But as I'm just following the current markup, this is something that I think needs a follow-up issue ticket.

However, the Button component should result in an `a` element when `isLink` is `true`.

All browsers should support wrapping block-level elements with `a`, so this should defiantly be done. But as I say — as a separate ticket.